### DISCUSSION TITLE
Find Your Mods

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#458] [Announcements]
### New feature(s)
* New option to skip the opening cutscene in the configuration editor menu  (@Raytwo / @jozz024)
* Merging support for bgm_property.bin files (@jozz024)
* Uncompiled lua files can now be loaded in place of compiled ones by renaming the extension to .lc (@itsmeft24)

### Improvement(s)
* Opening the mod manager on boot should now happen after being prompted to enable new mods and should properly display them as enabled
* ARCropolis is now noticeably smaller in size
* ARCropolis will now warn you if you are not on the latest version of the game until an issue is solved
* ARCropolis will now display a warning about deprecation if ``rom:/arc`` exists on the SD card. If you see the warning despite not using this method, consider deleting the directory.

### Bug fix(es)
* The prompt asking if you'd like to enable new mods found at boot time now works properly
* Users with stacked echo fighters should now have chara_1 display again for the echo (@Coolsonickirby)

### Removal(s)
* (Reminder) Support for rom:/arc merged modpack has been dropped

### Note
* If you go online with added slots, you take full responsibility on whatever happens to your Switch or Nintendo Online account.
* We do not endorse going online with added slots and we are only trying to make it more safe to not risk running into trouble with Nintendo. If you get banned, that's on you. If you get disconnected, that's also on you. Choose carefully.

# Read this
Skyline is not shipped with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v3.10.0'>Find Your Mods</a>.</em>

**Comment by AlexOnGitHubb** [comment]
Im sorry. Where is rom:/arc? I cant seem to find it. I dont recall installing it. Yet im still getting the crash error thing telling me to remove it.
I must be getting a false positive, but I dont know what you mean by "consider deleting the directory."

**Reply by Coolsonickirby** [reply]
`sd:/atmosphere/contents/01006A800016E000/romfs/arc`
